openstack/maia/templates/etc/_maia_scrape_config.yaml.tpl

```diff
@@ -96,6 +96,7 @@
       - '{__name__=~"^openstack_.+",project_id!=""}'
       - '{__name__=~"^limes_(?:project|domain)_(?:quota|usage)$"}'
       - '{__name__=~"^limes_swift_.+",project_id!=""}'
+      - '{__name__=~"^keppel_.+",project_id!=""}'
```

Comment:
Info to @viennaa and @notque: this looks way too inclusive, but it only matches a few hundred metrics per region.

Reply:
Provided it's keppel metrics with a project_id, it seems reasonable that those are available to the customer.

Reply:
I guess I should add we should look at what metrics exist, and add those to the documentation for expected metrics.

Reply:
Already done in cc/documentation-customer#355.